### PR TITLE
Sample for Multi-DC Persistence

### PR DESCRIPTION
**Pull request by patriknw**
TODO before merging:

* [ ] Tests
* [x] Sharding
* [x] Main class, including CassandraLauncher
* [x] HTTP routes
* [x] Example configuration, see https://github.com/akka/apps/blob/master/multidc/src/main/resources/application.conf
* [x] Protobuf serialization

### CODE REVIEW

File: akka-sample-persistence-dc-scala/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+1. In terminal 1: `sbt "runMain sample.persistence.multidc.ThumbsUpApp cassandra"`
+
+1. In terminal 2: `sbt "runMain sample.persistence.multidc.ThumbsUpApp 2551 eu-west"`
+
+1. In terminal 3: `sbt "runMain sample.persistence.multidc.ThumbsUpApp 2552 eu-central"`
+
+1. In terminal 4:
+    * To add a thumbs-up for resource `akka` from user `u1` in DC `eu-west`: `curl -X POST http://0.0.0.0:22551/thumbs-up/akka/u1`
+    * To add a thumbs-up for resource `akka` from user `u2` in DC `eu-west`: `curl -X POST http://0.0.0.0:22552/thumbs-up/akka/u2`
+    * To get the users that gave thumbs-up for resource `akka`: `curl http://0.0.0.0:22552/thumbs-up/akka`
```
Comment (johanandren) [inline akka-sample-persistence-dc-scala/README.md]: curl to `0.0.0.0`, does that really work? Would `127.0.0.1` be safer?

File: akka-sample-persistence-dc-scala/src/main/resources/application.conf [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+  persistence {
+    snapshot-store.plugin = "cassandra-snapshot-store"
+
+    multi-data-center {
+      all-data-centers = ["eu-central", "eu-west"]
+
+      speculative-replication.enabled = off
+
+//      cross-reading-replication {
```
Comment (johanandren) [inline akka-sample-persistence-dc-scala/src/main/resources/application.conf]: Add a first comment saying why this is commented out?

File: akka-sample-persistence-dc-scala/README.md (hunk 1 above)
Comment (patriknw) [reply akka-sample-persistence-dc-scala/README.md]: works for me, but  127.0.0.1 is better

**Comment by patriknw** [comment]
unrelated: https://github.com/akka/akka-samples/issues/51

**Pull request merged**